src/index.html

```diff
@@ -17,16 +17,29 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
-
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      action="#"
+      data-qa="big"
+      class="search__bar"
+    >
+      <input
+        type="text"
+        data-qa="keypress"
+        placeholder="Try “Los Angeles“"
+        class="search__bar--input input__big input__icon icon__big"
+      />
+    </form>
+    <form
+      action="#"
+      data-qa="small"
+      class="search__bar"
+    >
+      <input
+        type="text"
+        data-qa="hover"
+        placeholder="Try “Los Angeles“"
+        class="search__bar--input input__small input__icon icon__small"
```

Comment:
Similar to the previous comment, the class names 'input__small', 'input__icon', 'icon__small' should be revised to represent the meaning of the content.